java/ql/lib/ext/java.util.model.yml

```diff
@@ -424,7 +425,6 @@ extensions:
       # When `WithoutElement` is implemented, these should be changed to summary models of the form `Argument[this].WithoutElement -> Argument[this]`.
       - ["java.util", "Collection", "removeIf", "(Predicate)", "summary", "manual"]
       - ["java.util", "Iterator", "remove", "()", "summary", "manual"]
-      - ["java.util", "List", "clear", "()", "summary", "manual"]
       - ["java.util", "List", "remove", "(Object)", "summary", "manual"]
       - ["java.util", "Map", "clear", "()", "summary", "manual"]
       - ["java.util", "Set", "clear", "()", "summary", "manual"]
```

Comment:
With the current implementation we can't use `WithoutElement` and `WithElement` for anything else than collection content (not array content or dictionary content).
Would it maybe make more sense to include these as well for Java?

Reply:
WithoutElement should still apply as-is to `Set.clear()`, right?

Reply:
Yes, right - I will introduce that summary and make a test.

A question about `remove[If]` neutrals. Since we are not tracking individual elements for collections in Java, we will not be able to re-write these using `WithElement` and `WithoutElement`. Should we move these neutrals to their or should we have a summary instead (is it a better assumption that there is element flow via eg. `List.remove` as it only might remove a single element)?